packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/features/resolution/ResolutionFeature.java

```diff
@@ -144,6 +143,8 @@ static Size computeBestPreviewSize(int cameraId, ResolutionPreset preset)
    * @return The best possible {@link android.media.CamcorderProfile} that matches the supplied
    *     {@link ResolutionPreset}.
    */
+  @TargetApi(Build.VERSION_CODES.R)
+  @SuppressWarnings("deprecation")
```

Comment:
@camsim99 Apparently `@TargetApi()` on its own does not silence the deprecation warning? IIRC my original fix (before your changes for that edge case with EncoderProfiles) only added that annotation to get rid of the deprecation warnings, which did work back then. Strange that I did have to add it back here.